dataactcore/models/jobTrackerInterface.py

```diff
@@ -241,16 +240,20 @@ def getNumberOfRowsById(self,jobId):
         """ Get number of rows in file for job matching ID """
         return self.getJobById(jobId).number_of_rows
 
+    def getNumberOfValidRowsById(self, jobId):
+        """Get number of file's rows that passed validations."""
+        return self.getJobById(jobId).number_of_rows_valid
```

Comment:
Since we're allowing NULL values in the DB for `number_of_rows_valid`, wouldn't we need to have that check here to convert NULL results to 0?